egs/wsj/s5/steps/nnet3/get_egs.sh

```diff
@@ -265,23 +293,70 @@ valid_egs_opts="--left-context=$valid_left_context --right-context=$valid_right_
 
 echo $left_context > $dir/info/left_context
 echo $right_context > $dir/info/right_context
-num_pdfs=$(tree-info --print-args=false $alidir/tree | grep num-pdfs | awk '{print $2}')
-if [ $stage -le 3 ]; then
-  echo "$0: Getting validation and training subset examples."
-  rm $dir/.error 2>/dev/null
-  echo "$0: ... extracting validation and training-subset alignments."
 
+if ! $raw_nnet; then
```

Comment:
I was suggesting splitting the get_egs.sh script. Not the train script. If
you have time to modify the train script I would recommend adding the
changes to steps/nnet3/train_dnn.py.

On Wed, Mar 2, 2016 at 2:40 AM, Vimal Manohar <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/nnet3/get_egs.sh
> <https://github.com/kaldi-asr/kaldi/pull/539#discussion_r54688722>:
>
> >
> > +if ! $raw_nnet; then
>
> Agreed. What about steps/nnet3/tdnn/train.sh? Should I create a script
> like steps/nnet3/tdnn/train_raw.sh?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/539/files#r54688722>.
>